intellij-ide.md

```diff
@@ -136,9 +114,49 @@ html-based UI. For details see the [Observatory page](https://dart-lang.github.i
 To open Observatory:
 
 1. Run your app in debug mode.
-
 1. In the Debug panel at the bottom, click the **Stopwatch icon** ('Open Observatory').
 
+## Editing tips for Flutter code
+
+### Assists & Quick Fixes
+
+Assists are code changes related to a certain code identifier. A number of these
+are available when the cursor is placed on a Flutter widget identifier, as
+indicated by the yellow lightbulb icon. The assist can be invoked by clicking
+the lightbulb, or by using the keyboard shortcut `Alt-Enter`, as illustrated
+here:
+![IntelliJ editing assists](/images/intellij/assists.gif)
+
+Quick Fixes are similar, only they are shown with a piece of code has an error and they can assist in correcting it. They are indicated with a red lightbulb.
+
+We currently support the following assists & quick fixes:
+
+* 'Wrap new-expression with new widget': This can be used when you have a widget
```

Comment:
What if we rename both of these 'wrap' assists to 'Wrap with new widget'? Then you can combine the descriptions.